src/status_im/contexts/syncing/events.cljs

```diff
@@ -21,18 +21,44 @@
   {:events [:syncing/clear-states]}
   [{:keys [db]} role]
   {:db (dissoc db :syncing)})
+
 (defn- input-connection-string-callback
   [res]
   (log/info "[local-pairing] input-connection-string-for-bootstrapping callback"
             {:response res
              :event    :syncing/input-connection-string-for-bootstrapping})
-  (let [error (when (sync-utils/extract-error res)
-                (str "generic-error: " res))]
-    (when (some? error)
+  (let [response        (transforms/json->clj res)
+        installation-id (:installationId response)
+        key-uid         (:keyUID response)
+        error           (:error response)]
+    (when (seq installation-id)
+      (rf/dispatch [:syncing/set-syncing-installation-id installation-id key-uid]))
+    (when (seq error)
       (rf/dispatch [:toasts/upsert
                     {:type :negative
                      :text error}]))))
 
+(rf/defn initiate-pairing-process
+  {:events [:syncing/initiate-pairing-process]}
+  [{:keys [db]}]
+  {:db (assoc db :syncing/pairing-process-initiated? true)})
+
+(rf/defn set-syncing-installation-id
+  {:events [:syncing/set-syncing-installation-id]}
+  [{:keys [db]} installation-id key-uid]
+  {:db (assoc db
+              :syncing/key-uid         key-uid
+              :syncing/installation-id installation-id)})
+
+(rf/defn clear-syncing-data
```

Comment:
As you know, `rf/defn` is deprecated, and for new code it's the same effort rewriting it to use `reg-event-fx`. I remember a long time ago we agreed to avoid mixing in the same namespace `reg-event-fx` and `rf/defn`, but since that time, all CCs have been using `reg-event-fx`, so I think there's no need for us to grow usages of `rf/defn` anymore, unless in events mixed with `rf/merge`, but those are becoming more rare.